locksmith/src/utils/pricing.ts

```diff
@@ -275,7 +296,11 @@ export const createTotalCharges = async ({
       isCreditCardPurchasable: false,
     }
   }
-  const subtotal = Math.round(pricing.priceInAmount * 100)
+
+  // prioritize to credit card total if present
+  const price = creditCardTotal || pricing.priceInAmount
```

Comment:
`price` from for `createTotalCharges` and used for stripe charge, we use creditCardTotal as default or fallback to the converted one